### PR TITLE
fix: use accounts instead of prefixes for quoting

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

File: test/mocks/mock-plugin.js [hunk 1]
```diff
@@ -23,8 +22,8 @@ class MockPlugin extends EventEmitter {
     return this.connected
   }
 
-  getPrefix () {
-    return Promise.resolve(this.prefix)
+  getAccount () {
```
Comment (emschwartz) [inline test/mocks/mock-plugin.js]: Shouldn't this method be `getAddress` of `getAccount`?

Comment (sentientwaffle) [inline test/mocks/mock-plugin.js]: Right now its called [`getAccount`](https://github.com/interledger/js-ilp-plugin-bells/blob/ac0a7d6a11ade9e24df8e3eb57de6fc673e46134/src/lib/plugin.js#L253), (which is not in the spec) though there is a [proposal](https://github.com/interledger/rfcs/pull/62) to add `getAddress`. When that is merged we can rename it everywhere.

**Comment by emschwartz** [comment]
LGTM

**Comment by justmoon** [comment]
LGTM

**Pull request merged**